src/core/agent/services/identifierService.ts

```diff
@@ -68,6 +68,7 @@ class IdentifierService extends AgentService {
     "Cannot join multi-sig inception as we do not control any member AID of the multi-sig";
   static readonly UNKNOWN_AIDS_IN_MULTISIG_ICP =
     "Multi-sig join request contains unknown AIDs (not connected)";
+  static readonly KERIA_IS_DOWN = "The KERIA is down at the moment";
```

Comment:
`static readonly KERIA_CONNECTION_BROKEN = "The app is not connected to KERIA at the moment";`